cocos/audio/AudioEngine.cpp

```diff
@@ -241,7 +242,7 @@ int AudioEngine::play2d(const std::string& filePath, bool loop, float volume, co
             auto& audioRef = _audioIDInfoMap[ret];
             audioRef.volume = volume;
             audioRef.loop = loop;
-            audioRef.filePath = &it->first;
```

Comment:
I think change to use `std::string` for `AudioInfo::filePath` is better.
@PatriceJiang what's your opinion?

Reply:
And AudioInfo::profileHelper can change to `ProfileHelper` too.

Reply:
> I think change to use std::string for AudioInfo::filePath is better.
Yes